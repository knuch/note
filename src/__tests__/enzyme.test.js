import React from 'react';
import { shallow, mount } from 'enzyme';


const testElement = <div>
  <div>
    <p>shallow</p>
    <div>
      <div>
        <div>
          <p>deep</p>
        </div>
      </div>
    </div>
  </div>
</div>;
describe('Enzyme', () => {
  test('shallow works', () => {
    const wrap = shallow(testElement);
    expect(wrap).toMatchSnapshot();
  });
  test('mount works', () => {
    const wrap = mount(testElement);
    expect(wrap).toMatchSnapshot();
  });
});
