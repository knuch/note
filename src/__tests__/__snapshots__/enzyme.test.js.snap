// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Enzyme mount works 1`] = `
<div>
  <div>
    <p>
      shallow
    </p>
    <div>
      <div>
        <div>
          <p>
            deep
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Enzyme shallow works 1`] = `
<div>
  <div>
    <p>
      shallow
    </p>
    <div>
      <div>
        <div>
          <p>
            deep
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
